tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -22,7 +22,24 @@ public String name() {
   public void processFeature(SourceFeature sourceFeature, FeatureCollector features) {
     if (sourceFeature.canBeLine() && sourceFeature.hasTag("highway") &&
       !(sourceFeature.hasTag("highway", "proposed", "abandoned", "razed", "demolished", "removed", "construction"))) {
+
       String highway = sourceFeature.getString("highway");
+      String shield_text = sourceFeature.getString("ref");
+      String network_val = sourceFeature.getString("network");
+      shield_text = (shield_text == null ? null : shield_text.split(";")[0].replaceAll("\\s", ""));
```

Comment:
Shields don't look good on very long ref values (like `US 101;CA 1`), so we split on the `;` and because sprites generally should be smaller not bigger we also strip the internal whitespace.